### PR TITLE
Review

### PR DESCRIPTION
**Pull request by justynasadlak**

### CODE REVIEW

File: src/app/modules/shared/components/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import {Store} from '../../../../../store';
+
+@Component({
+  selector: 'app-toolbar',
+  templateUrl: './toolbar.component.html',
+  styleUrls: ['./toolbar.component.scss']
+})
+export class ToolbarComponent implements OnInit, OnDestroy {
+
+  isLogged$: Observable<boolean>;
```
Comment (angelikaMierzejewska) [inline src/app/modules/shared/components/toolbar/toolbar.component.ts]: this should be private

File: src/app/modules/shared/components/toolbar/toolbar.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  user: string;
```
Comment (angelikaMierzejewska) [inline src/app/modules/shared/components/toolbar/toolbar.component.ts]: this should be private

File: src/app/modules/user/login-dialog/login-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    this.loginForm = this.formBuilder.group({
+      login: ['', [Validators.required, Validators.maxLength(16)]],
+      email: ['', Validators.required],
+      password: ['', [Validators.required, Validators.minLength(8), Validators.pattern('^.*[A-Z]+.*$')]],
+      repeatPassword: ['', [Validators.required, Validators.minLength(8), Validators.pattern('^.*[A-Z]+.*$')]],
+      rememberMe: ['']
+    });
+  }
+
+  onSubmit() {
```
Comment (angelikaMierzejewska) [inline src/app/modules/user/login-dialog/login-dialog.component.ts]: types

File: src/app/modules/user/user-profile/user-profile.component.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  styleUrls: ['./user-profile.component.scss']
+})
+export class UserProfileComponent implements OnInit {
+
+  userData: UserData;
+
+  constructor(private userService: UserService) {
+  }
+
+  ngOnInit() {
```
Comment (angelikaMierzejewska) [inline src/app/modules/user/user-profile/user-profile.component.ts]: types

File: src/app/components/search-box/search-box.component.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  selector: 'app-search-box',
+  templateUrl: './search-box.component.html',
+  styleUrls: ['./search-box.component.scss']
+})
+export class SearchBoxComponent implements OnInit {
+
+  @Output()
+  search: EventEmitter<string> = new EventEmitter<string>();
+
+  searchForm: FormGroup;
```
[the remark this answers is not in the document]
Comment (adrianzaorski) [reply src/app/components/search-box/search-box.component.ts]: > searchForm should be private

Private properties can't be accessed from HTML template.

File: src/app/components/search-box/search-box.component.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<mat-card>
+  <mat-card-content>
+    <form [formGroup]="searchForm">
+      <mat-form-field>
+        <input type="text" placeholder="City" aria-label="Number" matInput [formControl]="cities"
```
Comment (adrianzaorski) [inline src/app/components/search-box/search-box.component.html]: Why aren't you using controls from `searchForm`, but instead you made `separated form-control`? Aria label should be `number`. By default, the input is `type="text"`, so there is no need of specifying this in that case.

File: src/app/components/search-box/search-box.component.ts [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      hotel: [''],
+      guests: [''],
+      checkIn: [''],
+      duration: ['']
+    }
+    );
+  }
+
+  ngOnInit() {
+    this.cities.valueChanges.subscribe(
```
[the remark this answers is not in the document]
Comment (adrianzaorski) [reply src/app/components/search-box/search-box.component.ts]: + type ngOnInit

**Pull request merged**